Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixSearchId.tsx

```diff
@@ -0,0 +1,280 @@
+                {this._renderInfoIcon()}
+                {this._renderSearchModal()}
+            </>
+        );
+    }
+
+    private _renderInfoIcon = () => {
+        return (
+            <a
+                className="bigfix-info"
+                onClick={() => {
+                    this._modalToggle();
+                }}
+            >
+                <i className="fas fa-info-circle ml-2" />
+            </a>
+        );
+    };
+
+    private _renderSearchModal = () => {
+        return (
+            <Modal
+                isOpen={this.state.searchModal}
+                toggle={this._modalToggle}
+                size="lg"
+                className="equipment-color"
+            >
+                <div className="modal-header row justify-content-between">
+                    <h2>Search Computer Id</h2>
+                    <Button color="link" onClick={this._modalToggle}>
+                        <i className="fas fa-times fa-lg" />
+                    </Button>
+                </div>
+
+                <ModalBody className="d-flex justify-content-center">
+                    {this._renderModalBody()}
+                </ModalBody>
+                <ModalFooter>{this._renderSearchButton()}</ModalFooter>
+            </Modal>
+        );
+    };
+
+    private _renderModalBody = () => {
+        return (
+            <Form className="w-75">
+                <FormGroup className="mb-5">
+                    <Label for="exampleSelect">Feild</Label>
+                    <Input
+                        type="select"
+                        name="select"
+                        id="exampleSelect"
+                        onChange={e => this._changeSelectedInput(e.target.value)}
+                        value={this.state.selectedFeild}
+                    >
+                        <option value="Name">Name</option>
+                    </Input>
+                </FormGroup>
+
+                <FormGroup>
+                    {this._renderInputSearch()}
+                    <FormFeedback>Computer name is required</FormFeedback>
+                </FormGroup>
+
+                {this._renderNameTable()}
+            </Form>
+        );
+    };
+
+    private _renderInputSearch = () => {
+        if (this.state.selectedFeild === "Name") {
+            return (
+                <>
+                    <label>Name</label>
+                    <Input
+                        type="text"
+                        name="name"
+                        id="computer name"
+                        placeholder="Enter Computer Name"
+                        invalid={this.state.valueToBeSearched.length < 1}
+                        onChange={e => {
+                            this.setState({
+                                valueToBeSearched: e.target.value
+                            });
+                        }}
+                    />
+                </>
+            );
+        } else if (this.state.selectedFeild === "Id") {
```

Comment:
I think Laura mentioned it, but typo on `selectedFeild`, should be `selectedField`